ShellScript/Bash.sublime-syntax

```diff
@@ -64,7 +64,8 @@ variables:
   is_path_component: (?=[^\s/]*/)
   is_start_of_arguments: '[`=|&;()<>\s]'
   is_variable: (?=\s*{{nbc}}(?:[({]{{nbc}}[)}])?{{nbc}}=)
-  keyword_boundary_end: (?!=)(?=[^\w_-]|$)
+  keyword_boundary_char: '[^-=\w]'
+  keyword_boundary_end: (?={{keyword_boundary_char}})
```

Comment:
Removing `$` from the match will cause issues at the end of the file where there is no newline character that could be matched by a negative set.

Reply:
Good catch, but can't add the `$` as it breaks other things. A negative lookahead does the same job though.